app/components/dashboard/manage/left-panel/component.js

```diff
@@ -85,6 +85,18 @@ export default Component.extend({
     this.set("fileBreadCrumbs", state.getCurrentFileBreadcrumbs()); // new collection, reset crumbs
   },
 
+  updateSessionData(listOfDataItems) {
+    console.log(listOfDataItems);
```

Comment:
Extraneous debug statement?

Reply:
The manage page is just a placeholder to test the modal (as per our discussion with @Xarthisius). The parent context is meant to do something with the listOfDataItems. But of course since this is just to test, manage will do nothing with the list. Just console logs for effect. Before merging this PR it would be best to remove the modal from the manage component.